uniffi_bindgen/src/bindings/swift/gen_swift.rs

```diff
@@ -220,23 +222,46 @@ mod filters {
     /// Lower a Swift type into an FFI type.
     ///
     /// This is used to pass arguments over the FFI, from Swift to Rust.
-    pub fn lower_swift(name: &dyn fmt::Display, _type_: &Type) -> Result<String, askama::Error> {
-        Ok(format!("{}.lower()", var_name_swift(name)?))
+    pub fn lower_swift(name: &dyn fmt::Display, type_: &Type) -> Result<String, askama::Error> {
+        match type_ {
+            Type::Duration => Ok(format!(
+                "{}.lower{}()",
+                var_name_swift(name)?,
+                type_swift(type_)?
```

Comment:
Stylistically, I think we should use `type_.canonical_name()` rather than `type_swift(type_)` here when including the name of the type as part of the name of something else. It won't matter in this case because `Duration` is a primitive type, but it would more clearly communicate the intent of the code, since that's what the canonical type name is for. So that would be something like:

```
format!(
    "{}.lower{}()",
    var_name_swift(name)?,
    type_.canonical_type_name()
)
```